rmgweb/database/views.py

```diff
@@ -794,7 +803,64 @@ def thermoData(request, adjlist):
     # Get the structure of the item we are viewing
     structure = getStructureInfo(molecule)
 
-    return render(request, 'thermoData.html', {'molecule': molecule, 'structure': structure, 'thermo_data_list': thermo_data_list, 'symmetry_number': symmetry_number, 'plotWidth': 500, 'plotHeight': 400 + 15 * len(thermo_data_list)})
+    return render(request, 'thermoData.html', {'molecule': molecule, 'structure': structure, 'thermo_data_list': thermo_data_list, 'symmetry_number': symmetry_number, 'ref_dict': ref_dict, 'word_list': word_list, 'plotWidth': 500, 'plotHeight': 400 + 15 * len(thermo_data_list)})
+
+
+def parseThermoComment(comment):
+    """
+    Takes a thermo comment (or any string) as input. Returns a dictionary whose keys 
+    correspond to groups or libraries exactly as they appear in the string, and whose values 
+    correspond to href links that direct to the specific library or group's database page.
+    """
+    
+    ref_dict = {}
+
+    # Search for library strings.
+    library_split_string  = comment.split("Thermo library: ")
+    gas_phase_substring = library_split_string[0].split()
+    if len(library_split_string) > 1: # if a match is found
+        if library_split_string[0] != '': 
+            try:
+                gas_phase_species = gas_phase_substring[-2]
+            except:
+                gas_phase_species = '' 
+        else:
+            gas_phase_species = ''
```

Comment:
> Thank you @jonwzheng. The work is awesome, and those demonstrations look good. Please see my minor comments.

Thanks Xiaorui. I'll add those in. I also think I need to squash these all into 1 commit and provide the specific exceptions in the `try / except` blocks instead of just excepting on any error.